cli/lsp/tsc.rs

```diff
@@ -4695,7 +4696,42 @@ fn op_script_names(state: &mut OpState) -> ScriptNames {
       .graph_imports_by_referrer(scope)
     {
       for specifier in specifiers {
-        script_names.insert(specifier.to_string());
+        let Ok(resolved) = state
+          .state_snapshot
+          .resolver
+          .as_cli_resolver(Some(scope))
+          .resolve(
```

Comment:
What kind/example of mapping is happening here? The graph imports are already supposed to be resolved

Reply:
When is that not handled in `npm_to_file_url()`?

Reply:
Oh sorry, I thought you were asking about the whole block (including the call to `npm_to_file_url`).

You're right, the 

```
.as_cli_resolver(Some(scope))
          .resolve(
```

call isn't needed. That part is already handled, I'll remove it